shared/widgets.py

```
@@ -1419,55 +1422,177 @@ def print_files_with_dialog(paths: list, parent=None, app_context=None) -> None:
-                    fallback.append(path)
-            else:
-                if not first:
-                    printer.newPage()
-                    page_rect = QRectF(painter.viewport())
-                first = False
-                img = QImage(path)
-                if not img.isNull():
-                    _draw_image_fitted(painter, img, page_rect)
-
-        painter.end()
+                    getattr(_pw, 'print')()  # QPrintPreviewWidget::print()
```

Ruff (0.15.10)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)